scripts/helpers.sh

```diff
@@ -9,3 +9,27 @@ get_tmux_option() {
         echo "$option_value"
     fi
 }
+
+get_capture_pane_start() {
+    split_direction=$(get_tmux_option "@extrakto_split_direction" "v")
+    split_size=$(get_tmux_option "@extrakto_split_size" "7")
+    grab_area=$(get_tmux_option "@extrakto_grab_area" "all")
+
+    capture_start="-32768"
```

Comment:
I've never asked, but where does this number come from? Should it not be `capture_start=-$(get_tmux_option "history-limit" "2000")` (as 2000 is [the default](https://github.com/tmux/tmux/blob/07a13697e10af0a4eee1c158cef679be177fb6e7/options-table.c#L230-L236))

Reply:
I think I got it from a sample on stackexchange.